src/Nest/CommonOptions/TimeUnit/Time.cs

```diff
@@ -307,5 +313,16 @@ private void Reduce(double ms)
 				Interval = TimeUnit.Millisecond;
 			}
 		}
+
+
+		private static string ExponentFormat(double d)
+		{
+			// Translate the double into sign, exponent and mantissa.
+			var bits = BitConverter.DoubleToInt64Bits(d);
+			// Note that the shift is sign-extended, hence the test against -1 not 1
+			var exponent = (int) ((bits >> 52) & 0x7ffL);
+			return new string('#', Math.Max(2, exponent));
+		}
```

Comment:
I just wondered if this conversion were a public method in framework somewhere , but looks like it isn't. Ignore 😄